.github/workflows/integration-test.yml

```diff
@@ -3,18 +3,24 @@ name: Integration test using mediamtx
 on: [push, pull_request]
 
 jobs:
-  build-and-test:
+  build-and-integration-test:
```

Comment:
Why do we need to change this file to add lint to CI?

Reply:
I can change it back now. This change was made originally because there were unit tests added on this PR to disambiguate which tests are being run. Now they are not being added.

Reply:
yeah lets change it back. This PR should only have lint in CI related changes.

Reply:
Changed